pom.xml

```diff
@@ -41,7 +41,22 @@
         <connection>scm:git:git://github.com/ulisesbocchio/spring-boot-security-saml</connection>
         <developerConnection>scm:git:git@github.com:ulisesbocchio/spring-boot-security-saml.git</developerConnection>
       <tag>spring-boot-security-saml-parent-1.0</tag>
-  </scm>
+    </scm>
+    
+    <repositories>
+      <repository>
+        <id>sonatype</id>
+        <url>https://oss.sonatype.org/content/repositories/releases/</url>
+      </repository>
+      <repository>
+        <id>mulesoft</id>
+        <url>http://repository.mulesoft.org/releases/</url>
+      </repository>
+      <repository>
+        <id>alfresco</id>
+        <url>https://artifacts.alfresco.com/nexus/content/repositories/public/</url>
+      </repository>
+    </repositories>
```

Comment:
is this a set of repos you use?

Reply:
Yes. It is the set of repositories for spring saml.